blogs/5_how_to_gorilla.html

```diff
@@ -0,0 +1,502 @@
+                        Moreover, we prioritize confidentiality and full user control, only executing commands with explicit permission from the user and never collecting output data. 
+                        <ol>
+                            <li>Get started with pip install: 
+                                <br><code>pip install gorilla-cli</code>>
+                            </li>
+                            <li> Using gorilla CLI to write to files:
+                                <ul>
+                                    <li>Command:
+                                    <br><code>$ gorilla “generate 100 random characters into a file called text.txt”</code>
+                                    </li>
+                                    <li>Output: Using arrow keys, toggle between commands and press <code>Enter</code>
+                                    <br><code>» cat /dev/urandom | env LC_ALL=C tr -dc 'a-zA-Z0-9' | head -c 100 > test.txt </code>
+                                    <br><code>echo $(head /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | dd bs=100 count=1) > test.txt </code>
+                                    <br><code>dd if=/dev/urandom bs=1 count=100 of=test.txt </code>
+                                    </li>
+                                </ul>
+                            </li>
+                            <li> Other Usage Examples:
+                                <ul>
+                                    <li>Command:
+                                    <br><code>$ gorilla “list all my GCP instances”</code>
+                                    </li>
+                                    <li>Output: 
+                                    <br><code>» gcloud compute instances list --format="table(name,zone,status)"</code>
+                                    <br><code>gcloud compute instances list --format table </code>
+                                    <br><code>gcloud compute instances list --format="table(name, zone, machineType, status)" </code>
+                                    </li>
+                                </ul>
+                            </li>
+                        </ol>
+                    </p>
+                    <h4 id="gorilla-inference"><a href="https://github.com/ShishirPatil/gorilla/tree/main/inference">Gorilla Inference 🦍 </a></h4>
+                    <p>
+                        <i>Note: If you would like to try out zero-shot Gorilla in a playground-like environment check out <a href="https://colab.research.google.com/drive/1DEBPsccVLF_aUnmD0FwPeHFrtdC0QIUP?usp=sharing">this Google Colab</a>. 
+                        </i>
+                        <br>
+                        <br>Running inference with gorilla is now supported on both hosted endpoints and locally! Inference with Gorilla can be done in 3 primary modalities: 
```

Comment:
Perhaps we should also have an example here of the chat completion API we have from the the colab. Basically to show OpenAI Chat completion API compliance and the change in `bash_url` required.